RELEASE-NOTES.md

```diff
@@ -1,5 +1,30 @@
 # Prefect Release Notes
 
+## 2.2.0
+
+### Exciting New Features 🎉
+* Added automatic detection of static arguments to `Task.map` in #6513
+* Using new orion-design flow-run-filter composition in #6483
```

Comment:
What does this mean to a user? Verb doesn't match other format.

Reply:
Looking closer at that PR it looks like it was actually just some code reorganization, so I just removed the bullet.